app/src/ui/history/selected-commit.tsx

```diff
@@ -302,11 +302,20 @@ export class SelectedCommit extends React.Component<
         <div className="panel blankslate">
           <img src={BlankSlateImage} className="blankslate-image" />
           <div>
-            <p>Unable to display diff when multiple commits are selected.</p>
+            <p>
+              Unable to display diff when multiple{' '}
+              {enableMultiCommitDiffs() ? 'non-adjacent ' : ' '}commits are
```

Comment:
This wording felt more common than `non-contiguous`; but completely open to feedback if there is a clearer way to communicate this. :)

Reply:
I like that better!